pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -1,34 +1,175 @@
 """
 Copyright 2020 Google LLC
-Copyright 2020 PerfectVIPs Inc.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-http://www.apache.org/licenses/LICENSE-2.0
+     http://www.apache.org/licenses/LICENSE-2.0
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-Regression script for RISC-V random instruction generator
 
 """
+import logging
+import riscv_core_setting
+from riscv_instr_pkg import *
+from riscv_core_setting import *
```

Comment:
Use specific imports
Avoid * imports

Reply:
As many fields are required from these files listing them will make code messy

Reply:
ok, will do that.